homework11/hw1.py

```diff
@@ -0,0 +1,58 @@
+"""
+Vasya implemented nonoptimal Enum classes.
+Remove duplications in variables declarations using metaclasses.
+
+from enum import Enum
+
+
+class ColorsEnum(Enum):
+    RED = "RED"
+    BLUE = "BLUE"
+    ORANGE = "ORANGE"
+    BLACK = "BLACK"
+
+
+class SizesEnum(Enum):
+    XL = "XL"
+    L = "L"
+    M = "M"
+    S = "S"
+    XS = "XS"
+
+
+Should become:
+
+class ColorsEnum(metaclass=SimplifiedEnum):
+    __keys = ("RED", "BLUE", "ORANGE", "BLACK")
+
+
+class SizesEnum(metaclass=SimplifiedEnum):
+    __keys = ("XL", "L", "M", "S", "XS")
+
+
+assert ColorsEnum.RED == "RED"
+assert SizesEnum.XL == "XL"
+"""
+
+
+class SimplifiedEnum(type):
+    def __new__(cls, name, bases, dct):
+        print(dct)
+        for key in dct.keys():
```

Comment:
try using dict comprehension.
attrbitues with dunder (__) in the begging can be got by "_{class_name}__keys"